specs/AdaptiveCardsXamarinSpec.md

```diff
@@ -0,0 +1,188 @@
+# AdaptiveCards Xamarin Renderer
+
+## Proposal Link
+
+There is no open issue for a Xamarin.Native package so I'm placing the Xamarin.Forms issue for now 
+https://github.com/microsoft/AdaptiveCards/issues/3255
+
+## Dependencies
+This project depends entirely on the Android and iOS renderer and their API's. This project is a Xamarin bindings library so the only new code to be included are the xamarin transformations for the underlying renderers. For generating the bindings, Xamarin requires the generated binaries (aar file for Android and framework file for iOS) to be included.
+
+### What is a bindings library
+
+A bindings library is an assembly that wraps the native library with C# wrappers so native code can be invoked via C# calls. Xamarin.Android implements bindings by using Managed Callable Wrappers (MCW) which are a JNI bridge that are used when managed code needs to invoke Java code. In the case of iOS, to bind a library an API definition file is needed; this is merely a C# source file that contains C# interfaces that have been annotated with a handful of attributes that help drive the binding. The API definition is merely the contract that will be used to generate the API.
+
+## Architecture
+
+As can be seen in the diagram below, the Android and iOS bindings libraries make use of the Java and Objective-C code that is already projected from the shared C++ code. The bindings also make use of the native APIs for both platforms yielding a C# library for every platform that can be consumed by any Xamarin platform.
+
+[image](assets/XamarinArchitecture.png)
```

Comment:
[Team Spec Review] Re: the image, We should rename the block currently listed as "C# projection" to be "C# object model"